Header header

string robot_name
uint8 robot_id
uint8 central_polygon
uint8 task
uint8 status           #0 Non defined, 1 defined, 2 running, 3 finished         	